terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperView.java

```diff
@@ -123,5 +125,11 @@ protected void prepareResponse(HttpServletRequest request,
         super.prepareResponse(request, response);
         response.setCharacterEncoding(StandardCharsets.UTF_8.name());
     }
-
+    
+    private String date2LocalDateString(final Date date) {
+        DateTimeFormatter fmt = DateTimeFormatter.ofPattern("uuuu/MM/dd");
```

Comment:
- [ ] DateTimeFormatter is thread-safe, so unless you are switching dynamically, you can cut it out to a static field.

Reply:
Static variables should be capitalized...

Reply:
It is fixed